### PR TITLE
2020-10-28 cass changelog blog post

### PR DESCRIPTION
**Pull request by loganloganlogan**
I've cleaned up the PR- let me know if there are any issues. @michaelsembwever

### CODE REVIEW

**Review by ErickRamirezDS — APPROVED** [review]
Thanks for making the changes @loganloganlogan. I've verified the following:
1. The correct 3 files (blog post in markdown + 2 header/footer images) are included in the PR.
2. The typo in the Tutorials section has been fixed.
3. The image URIs are correctly prefixed with `/img`.

Other remarks for posterity:
- I don't have a way of verifying that the HTLM code for the **Harry project** will render correctly.
- Content submissions are directed to `cassandra@constantia.io`. @michaelsembwever we should probably note this in the repo README so we know to update the blog pages should it change in the future.

Cheers! 🍻

File: src/_posts/2020-10-28-cass_changelog_1.markdown [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+[Open Source BI Tools and Cassandra](https://blog.anant.us/open-source-bi-tools-and-cassandra/) - Arpan Patel, Anant Corporation
+
+[Build Fault Tolerant Applications With Cassandra API for Azure Cosmos DB](https://dzone.com/articles/build-fault-tolerant-applications-with-cassandra-a) - Abhishek Gupta, Microsoft
+
+[Understanding Data Modifications in Cassandra](https://www.red-gate.com/simple-talk/blogs/understanding-data-modifications-in-cassandra/) - Sameer Shukla, Redgate
+
+## 
+![Cassandra Changelog footer](/img/changelog_footer.jpg "image_tooltip")
+## 
+Cassandra Changelog is curated by the community. Please send submissions to [cassandra@constantia.io](mailto:cassandra@constantia.io).
```
Comment (michaelsembwever) [inline src/_posts/2020-10-28-cass_changelog_1.markdown:92]: > Content submissions are directed to cassandra@constantia.io. @michaelsembwever we should probably note this in the repo README so we know to update the blog pages should it change in the future.

I don't know what the best approach is here. 

A separate mailing list I think would be best. For example media@cassandra.apache.org but I am open to suggestions to the name.  We can do this later on, we'd need some agreement on the dev ML first.
@loganloganlogan what are your thoughts on this?

**Comment by ErickRamirezDS** [comment]
Thanks @michaelsembwever for the [push to staging](https://cassandra.staged.apache.org/blog/2020/10/28/cass_changelog_1.html). I can confirm that the code for the Harry project is working. 👍

**Comment by michaelsembwever** [comment]
@loganloganlogan this is now deployed to production website.

**Pull request merged**